src/Administration/Resources/app/administration/src/module/sw-product/page/sw-product-detail/index.js

```diff
@@ -1069,6 +1081,8 @@ export default {
                 this.syncRepository
                     .save(this.product)
                     .then(() => {
+                        this.savePreferenceUnits();
```

Comment:
maybe check if the units has changed, otherwise save a request

Reply:
no, I mean to compare with the value we have in store :thonk:

Reply:
ah, you're right, in that case we need to create a data variable to store/hold the user's preferred units and use that data to do the further job, good point 👍🏻